data/json/items/comestibles.json

```diff
@@ -18174,6 +18174,34 @@
     },
     {
         "type" : "COMESTIBLE",
```

Comment:
Shouldn't be a comestible but a resource, like ethanol (check chemicals_and_resources.json).
Unless you intend to add a comestible effect to it.